tests/ha/test_server_pod_failure.py

```diff
@@ -125,8 +115,8 @@ def setup_method(self):
         LOGGER.info("Cluster status is online.")
         self.s3acc_name = f"ha_s3acc_{int(perf_counter_ns())}"
         self.s3acc_email = f"{self.s3acc_name}@seagate.com"
-        self.bucket_name = f"ha-mp-bkt-{self.random_time}"
-        self.object_name = f"ha-mp-obj-{self.random_time}"
+        self.bucket_name = f"ha-mp-bkt-{int(perf_counter_ns())}"
```

Comment:
this change might not be required if we are getting randomness

Reply:
self.random_time is we are setting once in setup method, 
perf_counter_ns() will always give a new number.


hence thought to update to make sure we always get new number